tests/designs/array_module/Makefile

```
@@ -42,6 +42,9 @@ COCOTB?=$(WPWD)/../../..
 ifeq ($(TOPLEVEL_LANG),verilog)
     VERILOG_SOURCES = $(COCOTB)/tests/designs/array_module/array_module.sv
 else ifeq ($(TOPLEVEL_LANG),vhdl)
+    ifeq ($(SIM),$(filter $(SIM),ius xcelium))
+       SIM_ARGS += -v93
```

nit: indent seems off here (and below)
```suggestion
        SIM_ARGS += -v93
```

---

Right, fixed now. I copied it from a makefile where the local convention was three spaces.